README.md

```diff
@@ -9,32 +9,33 @@ Docker EE products.
 Development
 ===========
 
-`docker/cli` is developed using Docker.
+`docker/cli` is developed using Docker. The `./tasks` script is used to run
```

Comment:
I'm confusing by what this means :sweat_smile:. Can it be simplified?

Something like:

> The `./tasks` script allows one to build and develop the `cli` via Docker images. For local development, the `Makefile` can be used directly with the same commands described below for `./tasks`.

This can probably be made better too though. cc @mstanleyjones ? :)